misc/python/materialize/checks/temporal_types.py

```diff
@@ -86,7 +86,7 @@ def validate(self) -> Testdrive:
 
 class TemporalPrecisionTypes(Check):
     def _can_run(self, e: Executor) -> bool:
-        return self.base_version >= MzVersion.parse("0.69.0-dev")
+        return self.base_version >= MzVersion.parse("0.70.0-dev")
```

Comment:
This change is unrelated. The nightly platform checks were failing because of this though, will probably make this a separate PR. This is because I think the time precision changes did not go out in the 0.69 version but will go in next week's version.

Reply:
I would make this a separate PR just so you can merge it independently of this change.

Reply:
Yeah, had pushed to check the build, creating a separate PR.

Reply:
Opened https://github.com/MaterializeInc/materialize/pull/21895